File: nelio-alves-algoritmo-e-logica-de-programacao/python/exercicios/medidas.py
A: float; B: float; C: float
areaQuadrado: float; areaTriangulo: float; areaTrapezio: float

A = float(input("Digite a medida A: "))
B = float(input("Digite a medida B: "))
C = float(input("Digite a medida C: "))

areaQuadrado = A * A
areaTriangulo = (A * B) / 2
areaTrapezio = ((A + B) * C) / 2

print(f"AREA DO QUADRADO = {areaQuadrado:.4f}")
print(f"AREA DO TRIANGULO = {areaTriangulo:.4f}")
print(f"AREA DO TRAPEZIO = {areaTrapezio:.4f}")

File: nelio-alves/algoritmo-e-logica-de-programacao/cpp/exercicios/medidas.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    double A, B, C, areaQuadrado, areaTriangulo, areaTrapezio;

    cout << "Digite a medida A: ";
    cin >> A;

    cout << "Digite a medida B: ";
    cin >> B;

    cout << "Digite a medida C: ";
    cin >> C;

    areaQuadrado = A * A;
    areaTriangulo = (A * B) / 2;
    areaTrapezio = ((A + B) * C) / 2;

    cout << fixed << setprecision(4);
    cout << "AREA DO QUADRADO = " << areaQuadrado << endl;
    cout << "AREA DO TRIANGULO = " << areaTriangulo << endl;
    cout << "AREA DO TRAPEZIO = " << areaTrapezio << endl;

	return 0;
}
